crates/spirv-builder/src/lib.rs

```diff
@@ -420,6 +420,9 @@ fn invoke_rustc(builder: &SpirvBuilder) -> Result<PathBuf, SpirvBuilderError> {
     let mut rustflags = vec![
         format!("-Zcodegen-backend={}", rustc_codegen_spirv.display()),
         "-Csymbol-mangling-version=v0".to_string(),
+        "-Zcrate-attr=no_std".to_string(),
```

Comment:
```suggestion
```
(as mentioned in my other review comments, this should be kept in the source of the crates, not here, because it's part of Rust not an implementation detail of Rust-GPU)